test/noauth.mk

```diff
@@ -22,13 +22,19 @@ test-noauth:
 # confirm that all the 'off' is respected).
 # Will install 2 control planes, one with only discovery + ingress ( micro ), one with galley, discovery, telemetry
 run-test-noauth-micro: install-crds
-	bin/iop ${ISTIO_NS}-micro istio-discovery ${BASE}/istio-control/istio-discovery ${IOP_OPTS} \
+	bin/iop ${ISTIO_CONTROL_NS}-micro istio-discovery ${BASE}/istio-control/istio-discovery ${IOP_OPTS} \
 		--set global.controlPlaneSecurityEnabled=false --set pilot.useMCP=false --set pilot.plugins="health"
-	kubectl wait deployments istio-pilot -n ${ISTIO_NS}-micro --for=condition=available --timeout=${WAIT_TIMEOUT}
+	kubectl wait deployments istio-pilot -n ${ISTIO_CONTROL_NS}-micro --for=condition=available --timeout=${WAIT_TIMEOUT}
 
-	bin/iop ${ISTIO_NS}-micro istio-ingress ${BASE}/gateways/istio-ingress --set global.istioNamespace=${ISTIO_NS}-micro \
+ifeq ($(ONE_NAMESPACE), 1)
+	bin/iop ${ISTIO_CONTROL_NS}-micro istio-ingress ${BASE}/gateways/istio-ingress --set global.istioNamespace=${ISTIO_CONTROL_NS}-micro \
 	 	${IOP_OPTS} --set global.controlPlaneSecurityEnabled=false
-	kubectl wait deployments ingressgateway -n ${ISTIO_NS}-micro --for=condition=available --timeout=${WAIT_TIMEOUT}
+	kubectl wait deployments ingressgateway -n ${ISTIO_CONTROL_NS}-micro --for=condition=available --timeout=${WAIT_TIMEOUT}
+else
```

Comment:
Especially now that I see how many of these `if ONE_NAMESPACE`, i think it may be a lot simpler to just change what the variables are as mentioned above

Reply:
Yes, good suggestion. Have updated namespace variables definition based on `ONE_NAMESPACE` env.